tests/integration/test_design.py

```diff
@@ -489,6 +504,14 @@ def test_named_selections(modeler: Modeler):
     design.delete_named_selection("OnlyCircle")
     assert len(design.named_selections) == 3
 
+    # Try to vefify name selection using earlier backend version
+    design = modeler.open_file(Path(FILES_DIR, "25R1BasicBoxNameSelection.scdocx"))
+    oldbackend = design._grpc_client._backend_version
+    design._grpc_client._backend_version = (24, 2, 0)
+    hello = design.named_selections
+    assert hello[0].faces == []
+    design._grpc_client._backend_version = oldbackend
```

Comment:
Oh... can we do this in a try...except..finally fashion so that we ensure that, even if the test fails, the proper _backend_version is restored (as part of the finally block)?

Another option is to use pytest fixtures to handle proper teardown/reset after test execution.

Ping me if you need more guidance for this =)